plexapi/video.py

```diff
@@ -169,11 +170,11 @@ def _loadData(self, data):
         self.viewedLeafCount = utils.cast(
             int, data.attrib.get('viewedLeafCount', NA))
         self.year = utils.cast(int, data.attrib.get('year', NA))
-        if self.isFullObject():
-            self.genres = [media.Genre(self.server, e)
-                           for e in data if e.tag == media.Genre.TYPE]
-            self.roles = [media.Role(self.server, e)
-                          for e in data if e.tag == media.Role.TYPE]
+        #if self.isFullObject(): # will be fixed with docs.
```

Comment:
Thanks! I havnt thought about that. I really should start on some tests. Im not sure the best way to to patch the http responses from pms.